deployment/samples/akri-video-streaming-app.yaml

```diff
@@ -47,3 +48,30 @@ spec:
     port: 80
     targetPort: 5000
   type: NodePort
+---
+apiVersion: v1
+kind: ServiceAccount
+metadata:
+  name: akri-video-streaming-app-sa
+---
+kind: ClusterRole
+apiVersion: rbac.authorization.k8s.io/v1
+metadata:
+  name: akri-video-streaming-app-role
+rules:
+- apiGroups: [""]
+  resources: ["services"]
+  verbs: ["list"]
+---
+apiVersion: rbac.authorization.k8s.io/v1
+kind: ClusterRoleBinding
+metadata:
+  name: akri-video-streaming-app-binding
+roleRef:
+  apiGroup: ""
+  kind: ClusterRole
+  name: akri-video-streaming-app-role
+subjects:
+  - kind: ServiceAccount
+    name: akri-video-streaming-app-sa
+    namespace: default
```

Comment:
First time creating rbac rules. @bfjelds do you know if this will work for all namespaces?

Reply:
i omitted the namespace in the helm rbac

Reply:
```yaml
apiVersion: v1
kind: ServiceAccount
metadata:
  name: akri-agent-sa
```

Reply:
oops ... wrong section

the helm stuff uses the helm install namespace:
```yaml
subjects:
  - kind: 'ServiceAccount'
    name: 'akri-controller-sa'
    namespace: {{ .Release.Namespace }}
```

Reply:
I omitted it in the ServiceAccount as well, but it did not let me drop it in the subjects.

Reply:
Ok, perhaps can be resolved in the integrate to Helm PR.